test/irb/test_cmd.rb

```diff
@@ -469,6 +469,17 @@ def test_show_source_method
       assert_match(%r[/irb\/init\.rb], out)
     end
 
+    def test_show_source_method_s
```

Comment:
Can we have more tests to cover:

- Multiple super level like `-sss`
- When the level of flags, like `-sss`, exceeds the actual super chain of the method
- When the super owner doesn't support `instance_method` method
- When `-s` is accidentally mixed with other characters, like `-sd`
    - I think in this case it should be seen as `-s`

Reply:
These are all great edge cases.  I'll get to work on implementing them today!

Reply:
Okay I added functionality and tests for all of these including `heredoc` for the code.  Let me know what you think!